lib/kafka.ts

```diff
@@ -1,11 +1,19 @@
 import { expect, Page } from '@playwright/test';
 import { config } from './config';
+import { closePopUp } from './popup';
 
 export const navigateToKafkaList = async function (page: Page) {
-  await page.getByRole('link', { name: 'Application and Data Services' }).click();
-  await page.getByRole('button', { name: 'Streams for Apache Kafka' }).click();
-  await page.getByRole('link', { name: 'Kafka Instances' }).click();
-  await expect(page.getByRole('heading', { name: 'Kafka Instances' })).toHaveCount(1);
+  if (!(await page.locator('[data-testid=router-link]', { hasText: 'Kafka Instances' }).isVisible())) {
```

Comment:
To navigate to KafkaInstances through side menu from everywhere. I think we don't have use case in main for it now. Do you think it would be better to have separate functions for it instead of one complex? This will help with maintainability for sure.

Reply:
> Do you think it would be better to have separate functions for it instead of one complex? This will help with maintainability for sure.

Yup, agreed 👍